### PR TITLE
Update AWS regex as per AWS Official Docs

### PR DESCRIPTION
**Pull request by LewisLebentz**
Updated AWS regex to match AWS's official access key regex search:

https://github.com/awslabs/git-secrets/blob/master/git-secrets#L238

### CODE REVIEW

**Comment by LewisLebentz** [comment]
Fixed merge conflict.

Hey guys @dxa4481 @KevinHock, any chance of getting this merged soon? Just resolved the conflict.

Have been using the regex in my own code since last year, and it works perfectly. Would just be easier if it was correct from the source!

**Comment by lcb** [comment]
Can someone merge this? LGTM :)

File: truffleHogRegexes/regexes.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "SSH (DSA) private key": "-----BEGIN DSA PRIVATE KEY-----",
     "SSH (EC) private key": "-----BEGIN EC PRIVATE KEY-----",
     "PGP private key block": "-----BEGIN PGP PRIVATE KEY BLOCK-----",
-    "Amazon AWS Access Key ID": "AKIA[0-9A-Z]{16}",
+    "Amazon AWS Access Key ID": "(A3T[A-Z0-9]|AKIA|AGPA|AIDA|AROA|AIPA|ANPA|ANVA|ASIA)[A-Z0-9]{16}",
     "Amazon MWS Auth Token": "amzn\\.mws\\.[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}",
     "AWS API Key": "AKIA[0-9A-Z]{16}",
```
Comment (lcb) [inline truffleHogRegexes/regexes.json]: Actually this line can be removed (duplicate)

**Comment by dxa4481** [comment]
Merged #13 which should cover this PR but feel free to re-open if I overlooked something

**Pull request closed without merging**